src/v0/form/css/complain.css

```diff
@@ -1414,41 +1414,46 @@ hr.pull-up {
 .issueselect label.block,
 #payday_questions_options label.block,
 #resolution_options label.block,
+.select-save-method label.block,
 .cr-radios label {
-    display: inline-block;
+    display: block;
     position: relative;
     background: #dee0e2;
     width: 100%;
-    border: 2px solid #dee0e2;
+    border: 1px solid #dee0e2;
     padding: 18px 30px 15px 45px;
     margin-bottom: 4px;
     margin-right: 4px;
     cursor: pointer;
+    -webkit-column-break-inside: avoid;
+    -moz-column-break-inside: avoid;
+    -o-column-break-inside: avoid;
+    -ms-column-break-inside: avoid;
```

Comment:
The build process is using autoprefixer so you may be able to remove all the vendor prefixes and only have `column-break-inside: avoid;` .